### PR TITLE
Fix #3456 Blank page is displayed if we enter number in the text editor in description field.

### LINKED ISSUE #3456: Blank page is displayed if we enter number in the text editor in description field
**Affected module**
UI

**Describe the bug**
If we enter numbers in the description edit box and try to edit it after saving, we are redirected to a blank page.

**To Reproduce**
- Navigate to Table details page from explore page
- Click on edit description
- Add number in description box and save the description
- Edit the description
- Blank page is displayed

https://user-images.githubusercontent.com/67048554/158780019-54b9b7b3-3243-4a38-a47f-431b6c5fc052.mp4

**Expected behavior**
Blank page should not be displayed

**Version:**
 - OpenMetadata version: 0.9.1

**Additional context**
This issue is happening where we have text editor present

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #3456 

### Describe your changes :

I worked on issue #3456 

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00 @vivekratnavel @harshach

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.tsx [hunk 1]
```diff
@@ -86,11 +81,8 @@ const RichTextEditor = forwardRef<editorRef, EditorProp>(
       getEditorContent(format: 'json' | 'markdown') {
         switch (format) {
           case Format.MARKDOWN:
-            return draftjsToMd(convertToRaw(editorState.getCurrentContent()));
-
-          case Format.JSON:
           default:
-            return convertToRaw(editorState.getCurrentContent());
+            return draftjsToMd(convertToRaw(editorState.getCurrentContent()));
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.tsx:85]: Remove switch case for now and add a comment for reference to use switch case if needed later

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
Minot comment, otherwise LGTM!

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.tsx:85]: @darth-coder00 , sure I will remove that.

**Review by darth-coder00 — APPROVED** [review]
Sweet!

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**